src/main/java/com/microsoft/aad/msal4j/AppMetadataCacheEntity.java

```diff
@@ -31,10 +31,13 @@
 import java.util.ArrayList;
 import java.util.List;
 
+/**
+ * Representation of application metadata.
```

Comment:
>Representation of  [](start = 3, length = 18)

Generally not sure this makes a lot of sense. What is this metadata about?

Reply:
Just has foci status. Not a public API, so removing java doc comment.